src/App.jsx

```diff
@@ -29,9 +35,20 @@ function App() {
 
       const data = await response.json();
 
+      //javascript sorting asc:
```

Comment:
No need to make a change for this review but make sure you are removing commented code before submitting PRs.